data/incipias/outfits.txt

```diff
@@ -0,0 +1,124 @@
+		"hull damage" 5
+		"minable damage" 6
+
+outfit "Tractor Beam"
+	category "Turrets"
+	mass 30
+	"outfit space" -30
+	"weapon capacity" -30
+	"turret mounts" -1
+	thumbnail "outfit/tractor beam"
+	licenses
+		"Hicemus Conflict"
+	weapon
+		sprite "projectile/tractor beam"
+			"frame rate" 6
+		"hardpoint sprite" "hardpoint/tractor beam"
+		sound "laser"
+		"hit effect" "tractor beam impact"
+		"inaccuracy" .5
+		"turret turn" 7.5
+		"velocity" 600
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 0.5
+		"firing heat" 3.2
+		# Deal damage to provoke ships.
+		"shield damage" 0.1
+		"hull damage" 0.1
+		"hit force" -40
+		"slowing damage" 0.2
+	description "The tractor beam is one of the few technologies of the Incipias that surpasses anything Humanity has made thus far - in fact, the first prototypes were built before the Incipias dreamed of space travel. Giant reactors were constructed to pull asteroids out of their home planet's orbit for easy access to more resources. Naturally, a mechanism to slow the asteroids down was also implemented to prevent accidental asteroid strikes."
+	description "As the Incipias' mastery of spaceflight grew, so did the generators' efficiency - this allowed for more compact, ship-based tractor beams. Resource collection was greatly streamlined by these new models - much to the benefit of the Incipias' otherwise sluggish ships."
+	description "Officially, these beams are used for mining operations; however, less scrupulous individuals have found they are equally-valuable in combat, using the beams to lock down and pull in their opponents for the kill."
+	description "The secret of how these beams work is only known to a few scientists of the Incipias who are under the control of the state. Such knowledge, if made public, could have disastrous consequences."
+
+effect "tractor beam impact"
+	sprite "effect/tractor impact"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"random angle" 2
+	"velocity scale" -.005
+
+
+outfit "Gas Class Shield"
+	category "Systems"
+	thumbnail "outfit/gas class shield"
+	mass 60
+	"outfit space" -60
+	"shield generation" .1
+	"energy generation" 4.2
+	"heat generation" 5
+	description "The shield technology is one of the most recent technologies of the Incipias. While the shield appears similar to a human shield device it works on a completely different base. In its core hydrogen is burned that provides energy for regeneration. But because scientists saw a more efficient use of this burning technology, the Incipias also made it the main power source of all of their ships."
+
+outfit "Liquid Class Shield"
+	category "Systems"
+	thumbnail "outfit/liquid class shield"
+	mass 80
+	"outfit space" -90
+	"shield generation" .25
+	"energy generation" 6.1
+	"heat generation" 6
+	description "This more advanced shield generator burns a liquid instead of a gas. This raises the efficiency by a lot but also requires some extra structures which make this outfit only popular on larger ships."
```

Comment:
```suggestion
	description "This more advanced shield generator burns a liquid instead of a gas. This raises the efficiency by a lot but also requires some extra structures, making this outfit popular only on larger ships."
```